### PR TITLE
Related to #266: Added data about emoji app for storelisting

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Partially done with #266 

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @vibhcool Please review

File: apps.json [hunk 1]
```diff
@@ -233,6 +233,33 @@
...
       "@type": "SoftwareApplication",
+      "permissions": [
+        "/api/search.json",
+      ],
+      "name": "emojiHeatmap",
+      "headline": "Emoji Heatmap",
+      "alternativeHeadline": "Emoji Heatmap with Loklak",
+      "applicationCategory": "Visualizer",
+      "applicationSubCategory": "Emoji",
+      "operatingSystem": "http://loklak.org",
```
Comment (vibhcool) [inline apps.json]: api.loklak.org

Comment (vibhcool) [inline apps.json]: I am a bit confused on operating system = api.loklak.org

File: emojiHeatmap/app.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "headline": "Emoji Heatmap",
+  "alternativeHeadline": "Emoji Heatmap with Loklak",
+  "applicationCategory": "Visualizer",
+  "applicationSubCategory": "Emoji",
+  "getStarted":"getStarted.md",
+  "appUse":"appUse.md",
+  "promoImage":"promo.png",
+  "appImages":["disp1.png", "disp2.png", "disp3.png"],
+  "oneLineDescription":"App which shows emoji usage in twitter",
+  "operatingSystem": "http://loklak.org",
```
Comment (vibhcool) [inline emojiHeatmap/app.json]: same

**Pull request merged**